rerun_cpp/src/rerun/entity_path.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <string_view>
+#include <vector>
+
+namespace rerun {
+    /// Construct an entity path by escaping each part of the path.
+    ///
+    /// For instance, `rerun::new_entity_path({"world", 42, "unescaped string!"})` will return
+    /// `"world/42/escaped\ string\!"`.
+    std::string new_entity_path(const std::vector<std::string_view>& path);
```

Comment:
I believe this should take a rerun collection. That way you can also call it with a single string view without overloading

Then again maybe that's a disadvantage because it muddles with the semantics of "list of path parts" 🤔

Reply:
There is plenty of room for improvement. I'd love to be able to call it with a mix of strings and integers to, and have the implementation call `to_string` on each non-string, but I feel this is good enough for now.